tiny_dnn/core/framework/tensor_storage.h

```diff
@@ -68,6 +68,20 @@ static inline size_t product(Container &c) {
                            std::multiplies<size_t>());
 }
 
+template <typename C1, typename C2>
+static inline size_t compute_offset(const C1 &c1, const C2 &c2) {
```

Comment:
not sure that we can reduce the operation since c1 (offset starting point) can have up to `kDimensions` and we have to iterate over c2 (current tensor shape) to compute the offset